cds/modules/previewer/templates/cds_previewer/macros/player.html

```diff
@@ -52,12 +56,19 @@
           },
           {% endfor %}
         ],
+        {% endif %}
         poster: '{{ obj.poster_uri }}',
         {% if obj.vr %}
         vr: {
-          360: true
+          360: true,
         },
         {% endif %}
+        {%- if embed_config.start -%}
+        startTime: parseInt({{ embed_config.start }}),
```

Comment:
why not using Python here, so you have a default value?
```
startTime: {{ embed_config.start | int(0) }},
endTime: {{ embed_config.end | int(<a number or duration if we now it>) }},
```
*warning, code not tested!*